shell/packages/sandstorm-db/migrations.js

```diff
@@ -312,6 +312,50 @@ function repairEmailIdentityIds() {
   });
 }
 
+function splitAccountUsersAndIdentityUsers() {
+  Meteor.users.find({"identities": {$exists: true}}).forEach(function (user) {
+    if (user.identities.length != 1) {
+      throw new Error("User does not have exactly one identity: ", user);
+    }
+    var identity = user.identities[0];
+    var identityUser = _.pick(user, "createdAt", "lastActive", "expires")
+    identityUser._id = identity.id;
+    identityUser.profile = identity.profile;
+    _.extend(identityUser, _.pick(identity, "unverifiedEmail"));
+    identityUser.profile.service = Object.keys(identity.service)[0];
+
+    // Updating this user needs to be a two step process because the `services` field typically
+    // contains subfields that are constrained to be unique by Mongo indices.
+    identityUser.stagedServices = _.omit(user.services, "resume");
+    if (identity.service.dev) {
+      identityUser.stagedServices.dev = identity.service.dev;
+    } else if (identity.service.email) {
+      identityUser.stagedServices.email = identity.service.email;
+    }
+
+    var accountUser = _.pick(user, "_id", "createdAt", "lastActive", "expires",
```

Comment:
Can we keep a complete copy of the user in a field called `oldUser` or something? Then delete it later, when we're sure we haven't lost anything.